doc/releases/migration-guide-4.0.rst

```diff
@@ -202,6 +202,22 @@ Controller Area Network (CAN)
 Display
 =======
 
+Disk
```

Comment:
Should it be part of `doc/releases/migration-guide-4.1.rst`? (v4.0.0-rc2 is tagged)

Reply:
I'm ok with that. This theoretically fixes a bug, but it is a large change to merge after RC2.

Reply:
Well, then let us add a bug label.